doc/conf.py

```diff
@@ -50,7 +50,10 @@
 ]
 
 
-intersphinx_mapping = {"photonics": ("https://strawberryfields.ai/photonics", None)}
+intersphinx_mapping = {
+    "photonics": ("https://strawberryfields.ai/photonics", None),
+    "xcc": ("https://xanadu-cloud-client.readthedocs.io/en/stable/", None)
```

Comment:
Does this work too? It would be nice to leave the `/ → /en/stable/` redirect up to Read the Docs.
```suggestion
    "xcc": ("https://xanadu-cloud-client.readthedocs.io/", None)
```

Reply:
I tried that first and it didn't work. Seems like it could find any Sphinx docs at the URL without the `/en/stable` part.

Reply:
Aw, okay. 😟

Reply:
Yep, this is required because readthedocs hosts _multiple_ docs versions (e.g., `stable`, `latest`, `v0.19.0`, etc.). So you need to be explicit about the version you want intersphinx to resolve links to

Reply:
Right, but I figured the redirect would just automatically choose one of these versions for us. The advantage of _not_ specifying a version is that changes to the default XCC Read the Docs version would be reflected in SF as well.